Makefile

```diff
@@ -1,22 +1,24 @@
+-include Makefile.devhelpers
+
 clean:
 	rm -rf venv
 	python3 -m venv venv
 
 
 install: clean
 	. venv/bin/activate && \
-	python setup.py install
+	python3 setup.py install
 
 
 install-requirements: install
 	. venv/bin/activate && \
-	pip install -r requirements-check.txt
+	pip3 install -r requirements-check.txt
 
 
 develop: clean
 	. venv/bin/activate && \
-	python setup.py develop && \
-	pip install -r requirements-check.txt
+	python3 setup.py develop && \
+	pip3 install -r requirements-check.txt 
```

Comment:
do you need the `3` sufix even after creating the venv with `python3`?

Reply:
hrm. probably not, once venv is loaded. this was mostly for kick-off under a rh7 dev env. i had also looked at updating the makefile a bit and realized it was adding some un-needed scope.. so more or less put things back, but stuck some helpers in a new makefile. i'll revert and push that up.

Reply:
reverted /3//